src/windows/ai_media_panel.py

```
@@ -85,6 +85,10 @@ def __init__(self, parent=None):
         # Create tab pages (Tags only – Analysis/Collections are backend-internal)
```

Ruff (0.15.18)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)